coalib/core/Core.py

```diff
@@ -212,8 +212,8 @@ def _schedule_bears(self, bears):
         """
         for bear in bears:
             if self.dependency_tracker.get_dependencies(
-                    bear):  # pragma: no cover
-                logging.warning(
+                    bear):
+                raise RuntimeError(  # Unreachable code
```

Comment:
Ya, using `close()` was the problem; changing it to `stop()` fixes it. https://github.com/jayvdb/coala/commit/4fa4fc94ef0be5abd8aac5c3d93bedf5ca0dc471